Pipfile.lock

```diff
@@ -132,73 +122,65 @@
             ],
             "version": "==1.2.2"
         },
-        "colorama": {
```

Comment:
Problem is we loose this lock unless we lock on windows.   I think if we add `colorama= {version = "*", markers="sys_platform == 'win32'"}` to the `Pipfile` and relock it would solve this.

Reply:
Ok. I'll add this to.

Reply:
Fixed.